.github/workflows/nodejs.yml

```diff
@@ -26,7 +35,12 @@ jobs:
           npm run lint
           npm run prettier-check
           npm run test-coverage
-          npm run test-dependent
```

Comment:
Doesn't seem like this is run from anywhere anymore, should we also remove the package.json script & dependency?

Reply:
Done 👍